### DISCUSSION TITLE
GrafanaCloud Install `sh` script not found

### DISCUSSION THREAD

**Discussion by frg100** [grafana-cold-storage/agent#3841] [Q&A] [answered]
https://raw.githubusercontent.com/grafana/agent/release/production/grafanacloud-install.sh returns a 404 error

Was this file moved?

**Comment by tpaschalis** [comment, marked as answer]
Yes, the script was removed in https://github.com/grafana/agent/pull/3485, since Grafana Cloud now has a different set of instructions for installing the Agent.

Have you noticed the script being in use anywhere? If so, let us know so we can fix it!

In the meantime, if you still want a copy you can take a look at the [release-v0.32](https://github.com/grafana/agent/blob/release-v0.32/production/grafanacloud-install.sh) branch.

**Comment by ebabeshko** [comment]
New link to script gives 403 now: https://storage.googleapis.com/cloud-onboarding/agent/scripts/grafanacloud-install.sh
